packages/clay-core/docs/treeview.mdx

```diff
@@ -295,7 +298,23 @@ Customizing the expander is possible using the `expanderIcons` property. Changin
 </TreeView>
 ```
 
-## Multiple Selection
+## Selection
+
+The selection allows the user to select one or more items in the TreeView, there are three main interactions that can be configured and are defined using the `selectionMode` prop.
+
+-   `single` select only one item.
+-   `multiple` select multiple items.
+-   `multiple-recursive` selects multiple items and the item's children recursively. The Parent item is also selected if all children are selected, if not all selected it is marked as intermediate.
```

Comment:
```suggestion
`multiple-recursive` selects multiple items and the item's children recursively. When all children are selected, the parent will be marked as selected, otherwise it will be marked as intermediate.
```